spec/takeaway_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'order'
```

Comment:
This should be split into a spec file for each class, and they shouldn't 'require' files other than the class they are testing.

Reply:
correct